docs/StardustDocs/topics/DataColumn.md

```diff
@@ -44,10 +44,22 @@ Special case of [`ValueColumn`](#valuecolumn) that stores other [`DataFrames`](D
 ```kotlin
 val age by column<Int>()
 
+// Access 3-rd cell in the "age" column of dataframe `df`.
+// This expression returns `Int` because variable `age` has `DataColumn<Int>` type.
```

Comment:
`age` has `ColumnReference<Int>` type, not `DataColumn<Int>`, because it doesn't have any data